jni/src/main/java/com/horizen/schnorrnative/ValidatorKeysUpdatesList.java

```diff
@@ -0,0 +1,207 @@
+package com.horizen.schnorrnative;
```

Comment:
Just a question: why passing maxPks to all the functions of the class ? It's a bit misleading. If it's for internal checks we can do them Java side and perhaps only in the constructor

Reply:
We could do that if you wish.

Reply:
Yeah, it's a bit misleading to me. Thanks !

Reply:
Done